documents/services/records/config.py

```diff
@@ -26,22 +26,18 @@
     is_published_record,
 )
 from oarepo_runtime.services.config.service import PermissionsPresetsConfigMixin
-from oarepo_runtime.services.files import FilesComponent
 from oarepo_runtime.services.records import pagination_links_html
 from oarepo_vocabularies.authorities.components import AuthorityComponent
 from oarepo_workflows.services.components.workflow import WorkflowComponent
 
-from common.services.config import FilteredResultServiceConfig
 from documents.records.api import DocumentsDraft, DocumentsRecord
 from documents.services.records.permissions import DocumentsPermissionPolicy
 from documents.services.records.results import DocumentsRecordItem, DocumentsRecordList
 from documents.services.records.schema import DocumentsSchema
 from documents.services.records.search import DocumentsSearchOptions
 
 
-class DocumentsServiceConfig(
-    PermissionsPresetsConfigMixin, FilteredResultServiceConfig
-):
+class DocumentsServiceConfig(PermissionsPresetsConfigMixin, RDMRecordServiceConfig):
```

Comment:
The FilteredResultServiceConfig has been removed from super - not sure if that is ok, as it changes the RecordList class